ui/templates/welcome.html

```diff
@@ -2,8 +2,17 @@
 {% block content %}
     <div class="section-header">
         <div class="row">
-            <div class="col-md-2 col-md-offset-5">{% if perms.learningresources.add_repository %}<a href="{% url 'create_repo' %}" role="button"
-                                                     class="btn btn-primary btn-large">Create repository</a>{% endif %}</div>
+            <div class="col-md-2 col-md-offset-5">
+                {% if perms.learningresources.add_repository %}
+                    <a href="{% url 'create_repo' %}"
+                       role="button"
+                       class="btn btn-primary btn-large">Create repository</a>
+                {% else %}
+                    <a href="mailto:mitx-support@mit.edu?subject=[LORE-app]Request%20permission%20to%20create%20repositories"
```

Comment:
d not understand why you hardcoded the email here and you did not take it from the settings.EMAIL_SUPPORT